crates/rolldown/tests/esbuild/loader/loader_file_one_source_two_different_output_paths_css/artifacts.snap

```diff
@@ -6,6 +6,9 @@ snapshot_kind: text
 
 ## assets/common-h8VKzHVD.png
 
+```png
+x
```

Comment:
Not sure if we want to show this in the snapshot file. The content seems meaningless.

Reply:
Only used for esbuild diff, by default it is hidden

Reply:
So this is for ensure the binary data generated by rolldown is the same as esbuild generated?

Reply:
yes

Reply:
https://github.com/rolldown/rolldown/blob/02a65e2d19aa38072d3009509d9bd2cd5adcd30e/crates/rolldown_testing/src/integration_test.rs#L255

> ensuring the binary data generated by rolldown is the same as esbuild generated?

I think current `String::from_utf8_lossy(bytes)` can't achieve that goal, it would replace invalid bytes, which makes the data dirty. It's better to use the raw data to do a strict equal.

Reply:
Maybe it's because esbuild didn't use a real picture. Well, since it's only in esbuild's test scope, no need to worry about it for now.